MSAL/src/instance/MSALDeviceInfoProvider.m

```diff
@@ -81,49 +78,46 @@ - (void)deviceInfoWithRequestParameters:(MSIDRequestParameters *)requestParamete
         fillDeviceInfoCompletionBlock(nil, nil);
         return;
     }
-    
-    if (@available(macOS 10.15, *))
+
+    // We are here means canCallSSOExtension is TRUE
```

Comment:
Here there seem to be a lot of changes but that's just re-indentation because the `if` above was removed,